app/controllers/home_controller.rb

```diff
@@ -1,6 +1,8 @@
 # Controller to handle root HTTP
 # connections and display the home page
 class HomeController < ApplicationController
+  layout 'sul-requests'
```

Comment:
(alternatively, should we just make this the `application.html.erb` layout file and let rails make it the default)